### ISSUE TITLE
TestClientCert* tests fails randomly

### ISSUE THREAD

**Issue by dmvolod** [infinispan/infinispan-operator#1202]
Tests are hangs with following message in the output
```
GET: https://172.18.0.5:32683/rest/v2/cache-managers/default/health
{"level":"info","ts":1626934362.4878182,"logger":"kubernetes_test","msg":"Received response for external address","response code":401}
```

Severer output from TestClientCertValidate
```
06:11:25,494 ERROR (SINGLE_PORT-ServerIO-3-1) [org.infinispan.rest.RestRequestHandler] ISPN012006: Uncaught exception in the pipeline io.netty.handler.codec.DecoderException: javax.net.ssl.SSLException: error:140E0197:SSL routines:SSL_shutdown:shutdown while in init
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:478)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:276)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.channel.ChannelInboundHandlerAdapter.channelRead(ChannelInboundHandlerAdapter.java:93)
	at org.infinispan.server.core.transport.StatsChannelHandler.channelRead(StatsChannelHandler.java:28)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
	at io.netty.channel.epoll.AbstractEpollStreamChannel$EpollStreamUnsafe.epollInReady(AbstractEpollStreamChannel.java:795)
	at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:480)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:378)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: javax.net.ssl.SSLException: error:140E0197:SSL routines:SSL_shutdown:shutdown while in init
	at org.wildfly.openssl.OpenSSLEngine.unwrap(OpenSSLEngine.java:625)
	at java.base/javax.net.ssl.SSLEngine.unwrap(SSLEngine.java:637)
	at io.netty.handler.ssl.JdkSslEngine.unwrap(JdkSslEngine.java:92)
	at io.netty.handler.ssl.JdkAlpnSslEngine.unwrap(JdkAlpnSslEngine.java:143)
	at io.netty.handler.ssl.SslHandler$SslEngineType$3.unwrap(SslHandler.java:284)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1358)
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1253)
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1300)
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:508)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:447)
	... 20 more
```

@ryanemerson could you please to have a look at this issue

**Comment by rigazilla** [comment]
seems that it's fixed by #1190

**Comment by dmvolod** [comment]
@rigazilla can we merge it into main to pass core test correctly?

**Comment by dmvolod** [comment]
But I tried to re-run #1192 test but it still fails on `main`

**Comment by rigazilla** [comment]
oh, right. I was using an old server image, let me try again

**Comment by rigazilla** [comment]
in my env it's working even with 12.1.7.Final. let's see CI result

**Comment by rigazilla** [comment]
Core tests is CI seem fine with #1190 and #1192 , @dmvolod you still have failures?

**Comment by dmvolod** [comment]
No, thanks @rigazilla I can see that CI is green now
